core/src/main/java/org/pentaho/agilebi/modeler/models/annotations/CreateMeasure.java

```diff
@@ -248,6 +250,8 @@ public boolean apply(
           measureMetaData.setName( getName() );
           measureMetaData.setDefaultAggregation( getAggregateType() );
           measureMetaData.setHidden( isHidden() );
+          measureMetaData.getMemberAnnotations().put( this.getName(),
```

Comment:
Maybe the key here should be a constant, and then further down instead of isEmpty() we use !contains.  What do you think?